src/extension.ts

```diff
@@ -72,9 +59,9 @@ async function registerKubernetesCloudProvider(): Promise<void> {
 
 export async function activate(extensionContext: ExtensionContext): Promise<unknown> {
     void WelcomePage.createOrShow();
-    await commands.executeCommand('setContext', 'isVSCode', env.uiKind);
+    void commands.executeCommand('setContext', 'isVSCode', env.uiKind);
     // UIKind.Desktop ==1 & UIKind.Web ==2. These conditions are checked for browser based & electron based IDE.
```

Comment:
Please remove this comment, it doesn't seem relevant anymore

Reply:
Removed